resview.py

```diff
@@ -636,6 +641,11 @@ def __call__(self, parser, namespace, values, option_string=None):
         'hide scalar bars',
     'position_vector':
         'define positions of plots [default: "0, 0, 1.6"]',
+    'position_vector2':
+        'define positions of plots [default: "0, 1.6, 0"]',
```

Comment:
Mayby add: ` - axis 1` and ` - axis 2`?

Reply:
I meant: `'define positions of plots along grid axis 1 [default: "0, 0, 1.6"]'` here in the help message.